mail-client/alpine/Manifest

```diff
@@ -1,3 +1,4 @@
+DIST 2.21.1-r1.tar.gz 7490140 BLAKE2B 13ac5ef43dd729a7cf21b44dcd6feace3e1813672aec537fd05d5ff90b67bec145ceef71cfcf35b0cb7e250bb307556f5dcf8ebb024bc6366fd0ace0a7c41161 SHA512 106caa7bfe86fd5e82554ba68b9ee579cd8495bcd0dfbf82d9d75da2df3bd292d42d67af8ec737091f2a820b2555c25bbf0058c9c7bb5de5f950b570ffc25c4d
```

Comment:
That's not a friendly or safe distfile name. It's asking for accidental collisions.

Reply:
Thanks for the hint, I fixed that.